packages/docusaurus/src/client/docusaurus.js

```diff
@@ -52,12 +54,12 @@ const docusaurus = {
 
     // Prefetch all webpack chunk assets file needed
     const chunkAssetsNeeded = chunkNamesNeeded.reduce((arr, chunkName) => {
-      const chunkAssets = window.__chunkMapping[chunkName] || [];
+      // eslint-disable-next-line no-undef
+      const chunkAssets = __webpack_require__.gca(chunkName) || [];
```

Comment:
For people reading this code for the first time, they won't know what `gca` stands for and what it does. I recommend adding some comments here to explain the "magic" and pointing them to our custom chunk asset plugin.